templates/internet-of-things/digital-signage.html

```diff
@@ -189,15 +224,29 @@ <h3 class="p-card__title">Hoxton Analytics</h3>
   </div>
 </div>
 
-<div class="p-strip--image is-dark is-deep" style="background-image: url('https://assets.ubuntu.com/v1/68b1a2e2-banner_pixels05.png');">
-  <div class="row">
-    <div class="col-8 col-start-large-3">
-      <h2>Build your own digital signage solution</h2>
-      <p>Ubuntu Core is entirely open source and supports a range of digital signage players. Just install it on a Raspberry Pi or NUC, download the app of your choice, create an account and you&rsquo;re ready to go.</p>
-      <p><a class="p-button--neutral js-invoke-modal" href="/internet-of-things/contact-us?product=digital-signage" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Internal Link', 'eventAction' : 'Digital Signage Page - Build your own', 'eventLabel' : 'Get in touch', 'eventValue' : undefined });">Get in touch</a></p>
+<section class="p-strip--light">
+  <div class="row u-equal-height">
+    <div class="col-4 u-align--left u-hide--small">
+      {{
+        image(
+          url="https://assets.ubuntu.com/v1/c4b290c8-Contact+us.svg",
+          alt="",
+          width="280",
+          height="280",
+          hi_def=True,
+        ) | safe
+      }}
+    </div>
+
+    <div class="col-8 u-vertically-center">
+      <div>
+        <h2>Let&rsquo;s work together</h2>
+        <p class="clear">Talk to us if you are thinking about using Ubuntu Core for your next Internet of Things project.</p>
```

Comment:
Can you remove `clear` it doesn't do anything.

Reply:
Good shout - I've updated and removed it from a couple of other places too.